cypress/e2e/awx/views/jobs.cy.ts

```diff
@@ -133,18 +133,18 @@ describe('job delete', () => {
         jobTemplate = jt;
 
         // Launch job to populate jobs list
-        cy.awxRequestPost(`/api/v2/job_templates/${jobTemplate.id.toString()}/launch/`, {}).then(
-          (jl) => {
-            jobList = jl;
-          }
-        );
+        // cy.awxRequestPost(`/api/v2/job_templates/${jobTemplate.id.toString()}/launch/`, {}).then(
```

Comment:
Why are we commenting out the launch? This will cause the[ test for relaunching a job](https://github.com/ansible/ansible-ui/blob/047379b537534d60c92bcee520aea74cb387230a/cypress/e2e/awx/views/jobs.cy.ts#L58) to fail.

Reply:
Upon review of the code, this launch is only called in the afterEach block of this spec file. It is superfluous.

Reply:
Isn't this in the `beforeEach`?